README.md

```diff
@@ -6,8 +6,8 @@ https://github.com/ddollar/foreman
 
 ## Getting Started
 
-    go get github.com/mattn/goreman
-
+    go install github.com/mattn/goreman@latest
+    
```

Comment:
nit: trim white spaces

Reply:
attempted to do so from GitHub UI, but it didn't register the whitespace change as sufficient for creating a new commit. apologies for littering 🗑️ ♻️ 🙏

Reply:
trimmed